superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -76,13 +85,32 @@ export type ExpandedControlPanelSectionConfig = Omit<
   controlSetRows: ExpandedControlItem[][];
 };
 
+const ActionButtonsContainer = styled.div`
+  ${({ theme }) => css`
+    display: flex;
+    position: sticky;
+    bottom: 0;
+    flex-direction: column;
+    align-items: center;
+    padding: ${theme.gridUnit * 4}px;
+    background: linear-gradient(transparent, white 25%);
```

Comment:
```suggestion
    background: linear-gradient(transparent, ${theme.colors.grayscale.light5} 25%);
```

Reply:
@michael-s-molina @geido this "25%" reminds me... I kind of think we should have a series of standard opacity stops and gradient stops in our theme.